config/routes.rb

```diff
@@ -384,6 +385,9 @@
   # this is a dumb thing
   get '/groups', to: 'dashboard#show'
 
+  # just to boot angular
```

Comment:
Arg, this has been bugging me for a while. I wonder if there's a way for us to _not_ have to define meaningless routes in beta just to allow people to show up on those pages in angular.

(Also could we take this comment out before merge?)